llvm/test/MC/AsmParser/sse2avx-intel.s

```diff
@@ -0,0 +1,80 @@
+# RUN: llvm-mc -triple x86_64-unknown-unknown -x86-asm-syntax=intel -msse2avx %s | FileCheck %s
```

Comment:
`-triple=x86_64` is sufficient and makes the line shorter

Reply:
Done